src/test/java/com/rultor/agents/github/ReleaseBinariesTest.java

```diff
@@ -0,0 +1,146 @@
+import com.rultor.spi.Profile;
+import com.rultor.spi.Talk;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.xembly.Directives;
+
+/**
+ * Tests for {@link ReleaseBinaries}.
+ *
+ * @author Krzysztof Krason (Krzysztof.Krason@gmail.com)
+ * @version $Id$
+ * @checkstyle ClassDataAbstractionCoupling (200 lines)
+ */
+public final class ReleaseBinariesTest {
+    /**
+     * Temp directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder temp = new TemporaryFolder();
+
+    /**
+     * ReleaseBinaries should attach artifact to release.
+     * @throws Exception In case of error
+     */
+    @Test
+    @Ignore
+    public void attachesBinaryToRelease() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final String tag = "v1.0";
+        final File dir = this.temp.newFolder();
+        final String target = "target";
+        final String name = "name-${tag}.jar";
+        final File bin = FileUtils.getFile(
+            dir.getAbsolutePath(), "repo", target, name.replace("${tag}", tag)
+        );
+        Files.createParentDirs(bin);
+        final byte[] content = RandomUtils.nextBytes(Tv.HUNDRED);
+        Files.write(content, bin);
+        final Talk talk = ReleaseBinariesTest.talk(issue, tag, dir);
+        new CommentsTag(repo.github()).execute(talk);
+        new ReleaseBinaries(
+            repo.github(),
+            new Profile.Fixed(
+                new XMLDocument(
+                    StringUtils.join(
+                        "<p><entry key='release'><entry key='artifacts'>",
+                        String.format("%s/%s", target, name),
+                        "</entry></entry></p>"
+                    )
+                )
+            )
+        ).execute(talk);
+        MatcherAssert.assertThat(
+            IOUtils.toByteArray(
+                new Releases.Smart(repo.releases()).find(tag)
+                    .assets().get(0).raw()
+            ),
+            Matchers.equalTo(content)
+        );
+    }
+
+    /**
+     * Make a talk with this tag.
+     * @param issue The issue
+     * @param tag The tag
+     * @param dir Daemon directory
+     * @return Talk
+     * @throws IOException If fails
+     */
+    private static Talk talk(final Issue issue, final String tag,
+        final File dir)
+        throws IOException {
+        final Talk talk = new Talk.InFile();
+        final String identifier = "id";
+        talk.modify(
+            new Directives().xpath("/talk")
```

Comment:
Are you sure all following nodes and attributes are relevant to the test?
I'm not sure about these:
```
.attr("later", Boolean.TRUE.toString())
.add("daemon").attr(identifier, "123")
.add("title").set("merge").up()
.add("script").set("empty").up()
.add("code").set("-7").up()
.add("href").set("http://test2").up()
.add("github-issue").set(Integer.toString(issue.number())).up()
```
These directives hide what is really matter in the test. As far as I understand, the test should work without all of them.
Correct me if I wrong